src/EditorFeatures/CSharpTest2/Recommendations/IsKeywordRecommenderTests.cs

```diff
@@ -167,5 +167,37 @@ public class IsKeywordRecommenderTests : KeywordRecommenderTests
             await VerifyAbsenceAsync(AddInsideMethod(
 @"var x = .$$0;"));
         }
+
```

Comment:
Probably not. How would you go about making sure there are enough characterization tests in place? I can add more, but I'm sure I'll miss some.